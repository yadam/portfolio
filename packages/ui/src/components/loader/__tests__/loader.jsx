import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '..';

jest.mock('../ripple.svg', () => 'ripple.svg');

it('renders correctly', () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
