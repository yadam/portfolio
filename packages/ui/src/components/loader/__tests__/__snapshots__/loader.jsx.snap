// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiCircularProgress-root-2 MuiCircularProgress-colorPrimary-5 MuiCircularProgress-indeterminate-4 Loader-root-1"
  role="progressbar"
  style={
    Object {
      "height": 40,
      "width": 40,
    }
  }
>
  <svg
    className="MuiCircularProgress-svg-7"
    viewBox="22 22 44 44"
  >
    <circle
      className="MuiCircularProgress-circle-8 MuiCircularProgress-circleIndeterminate-10"
      cx={44}
      cy={44}
      fill="none"
      r={20.2}
      strokeWidth={3.6}
      style={Object {}}
    />
  </svg>
</div>
`;
